project-plan.md

```diff
@@ -0,0 +1,189 @@
+- Proove ownership of a credential to a 3rd party.
+
+A limited set of ledgers and wallets should be investigated:
+
+**@TODO Make list of wallets to investigate**
+
+**@TODO Make list of ledgers to investigate**
+
+## 2.3 Oppgavebeskrivelse
+
+*See [project-description.md](./project-description.md)*
+
+### User story
+
+*Disclaimer: The BSc has no affiliation with Statens Vegvesen at the moment of writing, and the story below is pure fantasy.*
+
+The norwegian driver license issuer, Statens Vegvesen, is considering to start issuing it's driver licenses as verifiable credentials (VCs). Statens Vegvesen is not sure if verifiable credentials is the future yet, but are willing to try and dip it's toes in the water. Statens vegvesen will still issue credentials in the traditional way for the forseeable future.
+
+What Statens Veivesen want is a proof-of-concept SSI-application which will issue, hold and verify driver-licenses. They are hoping that this will bootstrap SSI in Norway, as this will enable other individuals and organizations to start experimenting with holding and verifying a serious credential which actually is useful.
+
+The proof-of-concept may demonstrate that a driver license as a verifiable credential could be considered on par, legally speaking, with traditional driver licenses.
+
+Statens Vegvesen wants the application to follow open standards which will enable the application to be agnostic about where and how credentials are issued, stored and verified. In other words Statens Vegvesen want to avoid the application to be locked to a specific ledger and a specific wallet.
+
+
+### Deliverables
+
+- Proof-of-concept SSI application which implements the user-story
+- A comparison matrix of ledger interoperability
+- A comparison matrix of wallet interoperability
+- Project report
+
+# 3. Prosjektorganisering
+
+## 3.1 Ansvarsforhold og roller
+
+| Navn        | Role                                                            |
+|-------------|-----------------------------------------------------------------|
+| Jonas       | Developer                  (Service Delivery Manager in Kanban) |
+| Snorre      | Client and domain expert 1 (Service Request Manager in Kanban)  |
+| Mariusz     | Domain expert 2                                                 |
+| Abylay      | Domain expert 3                                                 |
+| Deepti      | Academic supervisor                                             |
+
+## 3.2 Rutiner og regler i gruppa
+
+### Developer rules
+
+- Rule 1: To keep the project moving at all times, work on the project every day of the week. Minimum 5 minutes, maximum 8 hours.
+- Rule 2: Average 30 hours each week.
+- Rule 3: Respond to NTNU-mail, Discord-messages, Github-comments, Phone-calls, SMS in less than 24 hours.
+- Rule 4: Write meeting-notes for every meeting.
+- Rule 5: Log all hours in Toggl under these 5 categories: [Meeting, Organizing, Writing, Coding, Researching]
```

Comment:
WHere is this request coming from? Just personal insight?

Reply:
Yes :+1:  Explanation given in 3ddc188